cli/src/plz/cli/composition_operation.py

```diff
@@ -5,21 +5,19 @@
 
 
 class CompositionOperation(Operation):
-    def _run_composition(self,
-                         composition: dict,
+    def _run_composition(self, composition: dict,
```

Comment:
If the line is split, I find it easier to read if it's split on every argument. We can set `split_all_comma_separated_values` to achieve this.